### PR TITLE
Updated HiveOperator to use python3. Credit: Andy Yin

### PR DESCRIPTION
**Pull request by dimvor**

### CODE REVIEW

File: HiveOperator/README.md [hunk 1]
```diff
@@ -11,22 +11,21 @@ The operator runs on a custom Docker images that extends the SAP-deliver docker
...
 **1. Dockerfile**
-  - Extends the SAP-delivered Dockerfile com.sap.python27
+  - Extends the SAP-delivered Dockerfile `python:3.6.4-slim-stretch`
   - Imports the python library `pyhive`
   - Install Kerberos client libraries
 
 **2. Custom operator 'HiveOperator'**
-  - Derived from Pythin20Operator
-  - Uses image tags `python27:""` and `pyhive:pip2`
+  - Derived from Pythin3Operator
```
Comment (jensr84) [inline HiveOperator/README.md:27]: Looks like a typo "Pythin" and should by "Python", correct?

File: HiveOperator/README.md [hunk 2]
```diff
@@ -35,9 +34,9 @@ Before you start using the example, please make sure that:
 
 
 ## How to run
-  - Import [solution/HiveOperator-1.0.tgz](solution/HiveOperator-1.0.tgz) via `SAP Data Hub System Management` -> `Files` -> `Import Solution`
+  - Import [solution/HiveOperator-1.0.tgz](solution/HiveOperator-1.1.0.tgz) via `SAP Data Intelligence System Manager application` -> `Files` -> `Import Solution`
```
Comment (jensr84) [inline HiveOperator/README.md:37]: Correct name ist SAP Data Intelligence System **Management**

File: HiveOperator/src/vrep/vflow/dockerfiles/examples/HiveOperator/Dockerfile [hunk 3]
```diff
@@ -1,12 +1,23 @@
-FROM $com.sap.python27
+FROM python:3.6.4-slim-stretch
```
Comment (jensr84) [inline HiveOperator/src/vrep/vflow/dockerfiles/examples/HiveOperator/Dockerfile:1]: What is the reason to not derive from the pre-delivered base image $com.sap.sles.base, see https://help.sap.com/viewer/aff95eebc2e04c44816e6ff0d21c3c88/3.0.latest/en-US/d49a07c5d66c413ab14731adcfc4f6dd.html

**Review by jensr84 — CHANGES_REQUESTED** [review]

File: HiveOperator/src/vrep/vflow/dockerfiles/examples/HiveOperator/Dockerfile (hunk 3 above)
Comment (dimvor) [reply HiveOperator/src/vrep/vflow/dockerfiles/examples/HiveOperator/Dockerfile:1]: Because base image does not have apt-get

**Review by jensr84 — APPROVED** [review]

**Pull request merged**